src/com/serotonin/mango/vo/report/ReportChartCreator.java

```diff
@@ -58,6 +58,7 @@ public class ReportChartCreator {
      */
     private static final int IMAGE_WIDTH = 930;
     private static final int IMAGE_HEIGHT = 400;
+    private static final int IMAGE_HEIGHT_PX_INCREMENT_PER_POINT = 17;
```

Comment:
I said that the word Increment does not make sense here, it is the number of pixels per point label.